### PR TITLE
sql: update br docs about some sql options

### PR DESCRIPTION
**Pull request by RidRisR**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/17273
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-backup.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ BackupOption ::=
     "RATE_LIMIT" '='? LengthNum "MB" '/' "SECOND"
 |   "CONCURRENCY" '='? LengthNum
 |   "CHECKSUM" '='? Boolean
+|   "CHECKSUM_CONCURRENCY '='? LengthNum
```
Comment (lilin90) [inline sql-statements/sql-statement-backup.md:40]:
```suggestion
|   "CHECKSUM_CONCURRENCY" '='? LengthNum
```

File: sql-statements/sql-statement-restore.md [hunk 2]
```diff
@@ -41,6 +41,7 @@ RestoreOption ::=
     "RATE_LIMIT" '='? LengthNum "MB" '/' "SECOND"
 |   "CONCURRENCY" '='? LengthNum
 |   "CHECKSUM" '='? Boolean
+|   "CHECKSUM_CONCURRENCY '='? LengthNum
```
Comment (lilin90) [inline sql-statements/sql-statement-restore.md:44]:
```suggestion
|   "CHECKSUM_CONCURRENCY" '='? LengthNum
```

File: sql-statements/sql-statement-backup.md [hunk 3]
```diff
@@ -138,9 +139,15 @@ BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/'
 
 Use `RATE_LIMIT` to limit the average upload speed per TiKV node to reduce network bandwidth.
 
-By default, every TiKV node would run 4 backup threads. This value can be adjusted with the `CONCURRENCY` option.
+Before backup is completed, `BACKUP` would perform a checksum against the data on the cluster to verify correctness. 
 
-Before backup is completed, `BACKUP` would perform a checksum against the data on the cluster to verify correctness. This step can be disabled with the `CHECKSUM` option if you are confident that this is unnecessary.
+Use `CHECKSUM` to disable the check if you are confident that this is unnecessary.
+
+Use `CHECKSUM_CONCURRENCY` to controll the concurrency of checksumming in one table (default 4)
```
Comment (lilin90) [inline sql-statements/sql-statement-backup.md:146]:
```suggestion
Use `CHECKSUM_CONCURRENCY` to control the concurrency of checksum verification in a single table, and the default value is 4.
```

File: sql-statements/sql-statement-backup.md [hunk 4]
```diff
@@ -138,9 +139,15 @@ BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/'
...
+
+Use `CONCURRENCY` to specify the number of concurrent tasks BR can execute for backing up tables and indices. This parameter controls the thread pool size within BR, optimizing the performance and efficiency of backup operations.(default 4)
+
+One task represents one table range (or one index range) according to the backup schemas. If there is one table with one index.there will be two tasks to back up this table. This value should increase if you need to back up lots of tables or indices.
```
Comment (lilin90) [inline sql-statements/sql-statement-backup.md:150]:
```suggestion
Use `CONCURRENCY` to specify the number of concurrent tasks BR can execute for backing up tables and indices. This parameter controls the thread pool size within BR, optimizing the performance and efficiency of backup operations.

One task represents one table range or one index range, according to the backup schemas. For one table with one index, two tasks are used to back up this table. The default value of `CONCURRENCY` is 4. If you need to back up lots of tables or indices, increase its value.
```

File: sql-statements/sql-statement-restore.md [hunk 5]
```diff
@@ -129,9 +130,11 @@ RESTORE DATABASE * FROM 's3://example-bucket-2020/backup-05/'
 
 Use `RATE_LIMIT` to limit the average download speed per TiKV node to reduce network bandwidth.
 
-By default, TiDB node would run 128 restore threads. This value can be adjusted with the `CONCURRENCY` option.
+Before restore is completed, `RESTORE` would perform a checksum against the data from the archive to verify correctness. 
 
-Before restore is completed, `RESTORE` would perform a checksum against the data from the archive to verify correctness. This step can be disabled with the `CHECKSUM` option if you are confident that this is unnecessary.
+Use `CHECKSUM` to disable the check if you are confident that this is unnecessary.
+
+Use `CHECKSUM_CONCURRENCY` to controll the concurrency of checksumming in one table (default 4)
```
Comment (lilin90) [inline sql-statements/sql-statement-restore.md:137]:
```suggestion
Use `CHECKSUM_CONCURRENCY` to control the concurrency of checksum verification in a single table, and the default value is 4.
```

**Review by lilin90 — COMMENTED** [review]
@RidRisR PTAL at the comments and invite a tech reviewer.

**Review by YuJuncen — APPROVED** [review]

File: sql-statements/sql-statement-backup.md [hunk 6]
```diff
@@ -138,9 +138,13 @@ BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/'
 
 Use `RATE_LIMIT` to limit the average upload speed per TiKV node to reduce network bandwidth.
 
-By default, every TiKV node would run 4 backup threads. This value can be adjusted with the `CONCURRENCY` option.
+Before backup is completed, `BACKUP` would perform a checksum against the data on the cluster to verify correctness.
```
Comment (lilin90) [inline sql-statements/sql-statement-backup.md:141]:
```suggestion
Before the backup is completed, `BACKUP` would perform a checksum against the data on the cluster to verify correctness.
```

File: sql-statements/sql-statement-backup.md [hunk 7]
```diff
@@ -138,9 +138,13 @@ BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/'
 
 Use `RATE_LIMIT` to limit the average upload speed per TiKV node to reduce network bandwidth.
 
-By default, every TiKV node would run 4 backup threads. This value can be adjusted with the `CONCURRENCY` option.
+Before backup is completed, `BACKUP` would perform a checksum against the data on the cluster to verify correctness. 
 
-Before backup is completed, `BACKUP` would perform a checksum against the data on the cluster to verify correctness. This step can be disabled with the `CHECKSUM` option if you are confident that this is unnecessary.
+Use `CHECKSUM` to disable the check if you are confident that this is unnecessary.
```
Comment (lilin90) [inline sql-statements/sql-statement-backup.md:143]:
```suggestion
You can use the `CHECKSUM` parameter to disable the check if you are confident that this verification is unnecessary.
```

File: sql-statements/sql-statement-backup.md [hunk 8]
```diff
@@ -138,9 +138,13 @@ BACKUP DATABASE `test` TO 's3://example-bucket-2020/backup-05/'
...
+
+Use `CONCURRENCY` to specify the number of concurrent tasks BR can execute for backing up tables and indices. This parameter controls the thread pool size within BR, optimizing the performance and efficiency of backup operations.
```
Comment (lilin90) [inline sql-statements/sql-statement-backup.md:145]:
```suggestion
To specify the number of concurrent tasks that BR can execute for backing up tables and indices, use the `CONCURRENCY` parameter. This parameter controls the thread pool size within BR, optimizing the performance and efficiency of backup operations.
```

File: sql-statements/sql-statement-restore.md [hunk 9]
```diff
@@ -129,9 +129,9 @@ RESTORE DATABASE * FROM 's3://example-bucket-2020/backup-05/'
 
 Use `RATE_LIMIT` to limit the average download speed per TiKV node to reduce network bandwidth.
 
-By default, TiDB node would run 128 restore threads. This value can be adjusted with the `CONCURRENCY` option.
+Before restore is completed, `RESTORE` would perform a checksum against the data from the archive to verify correctness.
```
Comment (lilin90) [inline sql-statements/sql-statement-restore.md:132]:
```suggestion
Before the restore is completed, `RESTORE` would perform a checksum against the data in the backup files to verify correctness.
```

File: sql-statements/sql-statement-restore.md [hunk 10]
```diff
@@ -129,9 +129,9 @@ RESTORE DATABASE * FROM 's3://example-bucket-2020/backup-05/'
 
 Use `RATE_LIMIT` to limit the average download speed per TiKV node to reduce network bandwidth.
 
-By default, TiDB node would run 128 restore threads. This value can be adjusted with the `CONCURRENCY` option.
+Before restore is completed, `RESTORE` would perform a checksum against the data from the archive to verify correctness. 
 
-Before restore is completed, `RESTORE` would perform a checksum against the data from the archive to verify correctness. This step can be disabled with the `CHECKSUM` option if you are confident that this is unnecessary.
+Use `CHECKSUM` to disable the check if you are confident that this is unnecessary.
```
Comment (lilin90) [inline sql-statements/sql-statement-restore.md:134]:
```suggestion
You can use the `CHECKSUM` parameter to disable the check if you are confident that this verification is unnecessary.
```

**Review by hfxsd — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**